examples/gptreview-ghaction/README.md

```diff
@@ -0,0 +1,21 @@
+# GPTReview
+
+This folder contains an example of building and implementing your own code reviewer as part of GitHub Actions. 
+
+Below are the files present here:
+
+- `codereview.gpt`: Contains the GPTScript code and prompts.
+- `workflow.yaml`: The workflow file for the GitHub action.
+
+## Pre-requisites
+
+- GitHub Account
+- OpenAI API Key
+
+## How To Run This Example
+
+- Create a new repository in your GitHub account and create a `codereview.gpt` file in that repo based on the contents provided in this file.
+- Congfigure a GitHub Action for that repository and copy the contents from `workflow.yaml` to your `main.yaml`.
+- Configure your `OPENAI_API_KEY` and `GH_TOKEN` as environment variables.
```

Comment:
Can you provide links on how to set these as secrets and environment variables? I'm assuming this means in the GH repo so they are added at runtime with the Action?

Reply:
Added a link on how to setup environment variables for a repo.